README.md

```diff
@@ -58,15 +58,83 @@ For more information, read the [documentation](https://thelounge.github.io/docs/
 
 ## Development setup
 
-To run the app from source, just clone the code and run this in your terminal:
+
```

Comment:
This is waaaay too much information.
I don't think people need so much to run from source, and it's fair to assume people have a reasonable knowledge of Git when they want to contribute. If they don't, https://git-scm.com/book/en/v2 is an extremely well-written resource, and its 3 first chapters are probably enough to be 80%-git-fluent, and cover 100% of what's needed to contribute to a project on GitHub.